### ISSUE TITLE
Exercise 5 Level 5

### ISSUE THREAD

**Issue by swearnon** [nus-cs2030/2223-s2#83]
The instruction given is "By modifying level3.jsh, incorporate the above with minimal modifications to the file level5.jsh."

Does that mean that we can only alter the level3.jsh code and cannot change Service.java/Loader,java/ create a new class? Because given that there is a different output for the services by Recycled Loader, it seems modifying level3.jsh is insufficient.

**Comment by vanyabendatu** [comment]
You are allowed to modify other classes.

**Comment by 0liverkong** [comment]
your code should work for both level3.jsh and level5.jsh simultaneously, so the "Recycled Loader" should be an output of a toString() of a new class. (so there shouldnt be a need to modify loader.java)

**Comment by trwstin** [comment]
My man living in the future, already unlocked ex 5

**Comment by stxe09** [comment]
> My man living in the future, already unlocked ex 5

Haha oops

**Comment by timothysashimi** [comment]
You can modify other classes but you can do it without modifying other classes too!

**Comment by trwstin** [comment]
> would like to clarify if it is possible for us to create a new RecycledLoader class that inherits from Loader then?

yes, that is a possible solution

**Comment by Ramanenbh** [comment]
Is anyone else facing the same situation as myself, able to pass public test case for level5 but fail private test case?

**Comment by hhuilingg** [comment]
> Is anyone else facing the same situation as myself, able to pass public test case for level5 but fail private test case?

nope, i did not face that issue. but u may want to check if it is the issue on the recycled loader? it should be multiple of 3 instead of just 3. not sure if that's the issue 😓

**Comment by rshalinisai** [comment]
i modified the getServiceTime() to be = arrival time + service time + maintenance time --> I also created a maintenanceTime() method in my Loader class that returns 0 and in my RecycledLoader class that returns 60.
